content/python/concepts/strings/terms/count/count.md

```diff
@@ -13,31 +13,41 @@ CatalogContent:
   - 'paths/analyze-data-with-python'
 ---
 
-Takes in a list of values of any data type, and returns the number of times(count) a particular value is present within the list.
+The **`.count()`** method finds the number of times the specified substring occurs within a given string.
 
 ## Syntax
 
-```py
```

Comment:
This block of Python is never closed!

Reply:
@HishamT Are you looking at this on the mobile app? It shows everything as combined but all this was removed.

Reply:
No, I am on my computer. I can't see the closing, but I will take your word for it. :grin: